dataprofiler/profilers/profiler_options.py

```diff
@@ -1583,6 +1589,25 @@ def _numeric_stats_disabled_presets(self) -> None:
         self.set({"*.float.is_numeric_stats_enabled": False})
         self.set({"structured_options.text.is_numeric_stats_enabled": False})
 
+    def _memory_optimization_presets(self) -> None:
+        self.set({"structured_options.row_statistics.is_enabled": False})
```

Comment:
should `row_statistics` be enabled with HLL @taylorfturner

Reply:
that's more TBD IMO